cryptoki/tests/basic.rs

```diff
@@ -1003,3 +1003,28 @@ fn aes_gcm_with_aad() -> TestResult {
     assert_eq!(expected_cipher_and_tag[..], cipher_and_tag[..]);
     Ok(())
 }
+
+#[test]
+#[serial]
+fn get_slot_event() -> TestResult {
+    // Not implemented in SoftHSMv2
+    // https://github.com/opendnssec/SoftHSMv2/issues/370
+    let (pkcs11, _slot) = init_pins();
+    let event = pkcs11.get_slot_event().unwrap();
+    assert_eq!(None, event);
+    Ok(())
+}
+
+#[test]
+#[serial]
+fn wait_for_slot_event() -> TestResult {
+    // Not implemented in SoftHSMv2
+    // https://github.com/opendnssec/SoftHSMv2/issues/370
+    let (pkcs11, _slot) = init_pins();
+    let res = pkcs11.wait_for_slot_event();
+
+    match res {
+        Ok(_) | Err(Error::Pkcs11(RvError::FunctionNotSupported)) => Ok(()),
```

Comment:
:thinking: I think this entire `match` can be shortened to something like this:

```rust
assert!(matches!(res, Err(Error::Pkcs11(RvError::FunctionNotSupported))));
```

Reply:
I still included the error message. `assert_matches!(...)` would do that automatically but is "nightly-only".

Reply:
Yep, got it! 👍 Thanks!